colors/nord.vim

```diff
@@ -591,6 +592,30 @@ call s:hi("CocErrorSign" , s:nord11_gui, "", s:nord11_term, "", "", "")
 call s:hi("CocInfoSign" , s:nord8_gui, "", s:nord8_term, "", "", "")
 call s:hi("CocHintSign" , s:nord10_gui, "", s:nord10_term, "", "", "")
 
+" vim-go
+" > fatih/vim-go
+call s:hi("goBuiltins", s:nord7_gui, "", s:nord7_term, "", "", "")
```

Comment:
Please wrap this block between a `if exists("g:go_loaded_install") … endif`, like it's done in https://github.com/arcticicestudio/nord-vim/pull/294 ♥